bin/bench-exec-one.sh

```diff
@@ -52,6 +52,7 @@ bench_rts_opts_specific () {
     Prelude.Parallel/o-n-space/monad-outer-product/*) echo -n "-K4M -M256M" ;;
 
     Prelude.Rate/o-1-space/*) echo -n "-K128K" ;;
+    Prelude.Rate/o-1-space/asyncly/*) echo -n "-K128K" ;;
```

Comment:
Is this a part of this PR?

Reply:
the benchmark was failing, we can commit this separately if it keeps failing.